README.md

```diff
@@ -44,3 +44,50 @@ You can run any of them like so:
 ```
 npm start -- -f Nightly --pref=extensions.button-icon-preference_shield_mozilla_org.test.variationName=FB2L0
 ```
+
+## User Scenarios
+
+In all variations:
+
+  * Nothing different should happen in Private Browsing or Safe Mode operation
+  * If the user refreshes the page, by approximately the 6th time,
```

Comment:
On the 6th time it is guaranteed, but more likely it will happen on times 1-3. 40% chance the first time and increasing by 10% each time.